packages/core/src/queries/setting.ts

```diff
@@ -0,0 +1,25 @@
+import { Setting, SettingUpdate, Settings } from '@logto/schemas';
+import { sql } from 'slonik';
+
+import pool from '@/database/pool';
+import { buildUpdateWhere } from '@/database/update-where';
+import { convertToIdentifiers, OmitAutoSetFields } from '@/database/utils';
+
+export const defaultSettingId = '0000000000';
```

Comment:
will it be better to use a declarative string?
```suggestion
export const defaultSettingId = 'default';
```